server/pulp/server/webservices/views/content.py

```diff
@@ -167,7 +170,7 @@ def post(self, request):
 
         :raises: OperationPostponed when an async operation is performed
         """
-        all_orphans = request.body_as_json
+        all_orphans = request.body_as_json or {}
```

Comment:
I have actually just realized that this conditional will have to be used for every instance where we allow an empty body, so I am going to change this do be the default in the util rather than in each use.

`s/request.body_as_json = None/request.body_as_json = {}/`